src/Parsec/Shaiya/Monster/DBMonsterTextRecord.cs

```diff
@@ -9,6 +9,6 @@ public sealed class DBMonsterTextRecord : IBinarySDataRecord
     public long Id { get; set; }
 
     [ShaiyaProperty]
-    [LengthPrefixedString(false)]
+    [LengthPrefixedString(true)]
```

Comment:
Unless you found a translation file where the strings are null terminated (suffixed with `\0`), this is wrong. All of the BinarySData translation file texts are not null terminated.

The attribute `LengthPrefixedString` has this constructor:
```cs
    public LengthPrefixedStringAttribute(bool includeStringTerminator)
    {
        IncludeStringTerminator = includeStringTerminator;
    }
```

So, `[LengthPrefixedString(false)]` means that the string is length prefixed but it's not null-terminated.

Reply:
Yeah, but inside EG we can see that they are using terminator, but possible that it's not required for others..
![image](https://user-images.githubusercontent.com/20420491/233543030-7b854541-5cd9-4f78-bd99-2c6f0f692bab.png)